tests/testthat/test-variable-catalog.R

```diff
@@ -99,6 +98,16 @@ with_mock_crunch({
                 type=c("numeric", "categorical", "multiple_response")
             )))
     })
+    
+    test_that("Variable Catagory as.data.frame method", {
+      expect_equal(as.data.frame(varcat[1:3]),
+                    data.frame(
+                      alias=c("birthyr", "gender", "mymrset"),
+                      name=c("Birth Year", "Gender", "mymrset"),
+                      type=c("numeric", "categorical", "multiple_response"),
+                      stringsAsFactors = FALSE
+                    ))
+         })
 })
```

Comment:
It would be nice to have a test for the failure states if a user passes fields that aren't in the catalog, etc.